### DISCUSSION TITLE
Domain verification issue

### DISCUSSION THREAD

**Discussion by Oludefiyinfoluwa06** [vercel/community#6457] [Help] [answered]
### Summary

I made use of a custom domain in my account and deleted it from the project I used it for. Then I tried to use it again in another project, it now says that it's being used in another account. How can you help me solve this problem?

### Steps to Reproduce

![Screenshot (105)](https://github.com/vercel/community/assets/120565527/a9507f3a-2436-45cb-b36b-cdf1ba27b6ad)

**Comment by amyegan** [comment, marked as answer]
Hey @Oludefiyinfoluwa06. Your team may have been recently updated to our new account structure. In that case your domain may be associated with your personal account instead of your new hobby team. You can learn more about the change and how to update your domain in https://github.com/vercel/community/discussions/6298

**Reply by Oludefiyinfoluwa06** [reply]
Thank you, problem's solved

[a comment by an automated account is not shown]
